lib/travis/build/addons/ssh_known_hosts.rb

```diff
@@ -25,6 +25,7 @@ def config
           def add_ssh_known_hosts
             sh.fold 'ssh_known_hosts.0' do
               sh.echo "Adding ssh known hosts", ansi: :yellow
+              sh.cmd 'mkdir -p "${TRAVIS_HOME}/.ssh"'
```

Comment:
We should be more uniform in use of certain methods, but here, we can use:

```ruby
sh.mkdir "${TRAVIS_HOME}/.ssh", recursive: true
```

Reply:
Sure. Didn't think of it since some code below used a raw command for mkdir.

Reply:
Right. We should do better with respect to the uniformity throughout the code base.